### PR TITLE
Feature/circom parser

### PR DESCRIPTION
**Pull request by Hrom131**
- [ ] Since this PR suggests a **bug fix**, the relevant tests have been added.
- [x] Since this PR introduces a **new feature**, the update has been discussed in an Issue or with the team.
- [ ] This PR is just a **minor change**, like a typo fix.

---

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,9 +50,10 @@
     ]
   },
   "dependencies": {
+    "@distributed-lab/circom-parser": "0.1.0",
```
Comment (Arvolear) [inline package.json:53]:
```suggestion
    "@distributedlab/circom-parser": "0.1.0",
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solarity/hardhat-zkit",
-  "version": "0.4.0-rc.0",
+  "version": "0.4.0-rc.1",
```
Comment (Arvolear) [inline package.json:3]:
```suggestion
  "version": "0.4.0",
```

File: README.md
[the remark this answers is not in the document]
Comment (Hrom131) [reply README.md:1]: Let's go

**Review by Arvolear — APPROVED** [review]
LGTM!

**Pull request merged**